### PR TITLE
Use official way of installing cmtcl

### PR DESCRIPTION
**Pull request by rabi**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running

### CODE REVIEW

File: roles/cert_manager/tasks/validate_certs.yml [hunk 1]
```diff
@@ -21,11 +21,10 @@
     mode: '0755'
 
 - name: Install cert-manager cmctl CLI
-  ansible.builtin.unarchive:
-    src:
-      "https://github.com/cert-manager/cert-manager/releases/{{ cifmw_cert_manager_version }}/download/cmctl-linux-amd64.tar.gz"
-    dest: "{{ lookup('env', 'HOME') }}/bin/"
-    remote_src: true
+  ansible.builtin.shell: |
```
[the remark this answers is not in the document]
Comment (rabi) [reply roles/cert_manager/tasks/validate_certs.yml:24]: that sounds better thanks. Let's see if it works.

File: roles/cert_manager/tasks/validate_certs.yml [hunk 2]
```diff
@@ -21,11 +21,14 @@
...
 
 - name: Install cert-manager cmctl CLI
-  ansible.builtin.unarchive:
-    src:
-      "https://github.com/cert-manager/cert-manager/releases/{{ cifmw_cert_manager_version }}/download/cmctl-linux-amd64.tar.gz"
-    dest: "{{ lookup('env', 'HOME') }}/bin/"
-    remote_src: true
+  become: true
+  vars:
+    _os: "{{ lookup('pipe', 'go env GOOS') }}"
```
Comment (cjeanner) [inline roles/cert_manager/tasks/validate_certs.yml:26]: humpf, we need to mock that lookup I think - the pod doesn't have `go` - hence the lookup is failing. At least that's my understanding of the issue. it means editing `.ansible-lint` to extend the mocked modules (?) by adding
`lookup_plugin.pipe`...

**Review by rlandy — COMMENTED** [review]
approving

File: roles/cert_manager/tasks/validate_certs.yml [hunk 3]
```diff
@@ -20,12 +20,18 @@
     state: directory
     mode: '0755'
 
-- name: Install cert-manager cmctl CLI
-  ansible.builtin.unarchive:
-    src:
-      "https://github.com/cert-manager/cert-manager/releases/{{ cifmw_cert_manager_version }}/download/cmctl-linux-amd64.tar.gz"
-    dest: "{{ lookup('env', 'HOME') }}/bin/"
-    remote_src: true
+# jinja[invalid] is triggered by the call to the lookup_plugin.pipe
```
Comment (rabi) [inline roles/cert_manager/tasks/validate_certs.yml:23]: Thanks! I guess it would have been better to use shell module like I had proposed initially.

**Pull request merged**